packages/neptune-extension/package.json

```diff
@@ -7,17 +7,24 @@
   "author": "Neptune Labs",
   "main": "dist/labextension/neptune-notebook.js",
   "scripts": {
-    "check-types": "tsc",
-    "check-types-watch": "tsc --watch",
```

Comment:
I'm afraid that because because of 'platform' alias there is no option to run tsc in "watch" mode against the whole codebase.